### ISSUE TITLE
What if I delete a plan with active subscriptions?

### ISSUE THREAD

**Issue by htmgarcia** [OSTraining/Simplerenew#335]
I was thinking in a warning to explain the plan you are about to delete has active users.

**Comment by billtomczak** [comment]
Yeah - Recurly lets you do that. At some point I added some code to cover for that. Subscriptions are still okay, but who knows what happens when renewal time comes around!

Unfortunately, I haven't found a way to query the API to test this.

**Comment by stevejburge** [comment]
Can we add a column in the Joomla admin showing the number of active subscribers on the plan?

**Comment by billtomczak** [comment]
Not right now. One of the pro features we've discussed is subscriptions without direct payment like we're doing now with cbsubs.

Once I finish groupleaders (almost there!) that will be the next thing I start to tackle with SR.
